docs/transparent-validation/msi-z690/laboratory-assembly-guide.md

```diff
@@ -361,3 +368,16 @@ the below-described steps:
 
 1. If the reading was successful, the output from the command above should
     contain the phrase `Verifying flash... VERIFIED`.
+
+### Reading power status
+
+To read the current power status use the following command:
+
+```bash
+cat /sys/class/gpio/gpio12/value
+```
+
+Output meaning:
```

Comment:
That section should be named `Example output` and indicate what the given output means.

Reply:
Applied